packages/plugingen/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@alchemy/6900-plugingen",
```

Comment:
TODO: what should we call this package?

Reply:
I think plugingen is good enough tbh both for the package and the CLI. it describes exactly what it's going to do